guides/source/7_1_release_notes.md

```diff
@@ -150,7 +150,51 @@ and reporting of the bulk enqueuing process.
 
 ### Composite primary keys
 
-TODO: Add description
+Applications can now create database tables with composite primary keys, and Rails will be able to derive these keys directly from the schema. This feature is particularly beneficial for many-to-many relationships and other complex data models where a single column is insufficient to uniquely identify a record.
```

Comment:
Rails was previously able to create database tables with CPKs, right? 🤔 What do you think about highlighting that it's a change in CPK support at the model level, e.g.
```suggestion
Composite primary keys are now supported at both the database and application level. Rails is able to derive these keys directly from the schema. This feature is particularly beneficial for many-to-many relationships and other complex data models where a single column is insufficient to uniquely identify a record.
```

Reply:
True, I like your suggestion, will change it